samples/ingestion/ingestion-client/Setup/ArmTemplateBatch.json

```diff
@@ -194,15 +194,22 @@
             }
         },
         "PiiRedaction": {
-            "defaultValue": "None",
-            "type": "String",
-            "allowedValues": [
-                "None",
-                "UtteranceAndAudioLevel"
-            ],
-            "metadata": {
-                "description": "A value indicating whether personally identifiable information (PII) redaction is requested. Will only be performed if a Text Analytics Key and Region is provided."
-            }
+          "defaultValue": "None",
+          "type": "String",
+          "allowedValues": [
+              "None",
+              "UtteranceAndAudioLevel"
+          ],
+          "metadata": {
+              "description": "A value indicating whether personally identifiable information (PII) redaction is requested. Will only be performed if a Text Analytics Key and Region is provided."
+          }
+        },
+        "ConversationSummarizationAspects": {
```

Comment:
there was a discussion how we should expose conversational features as part of the ARM template, and the conclusion was that we should expose them only as part of the variables, but not the parameters (as long as the nuget is still a beta version).
Afterwards, we can add them to the params, and think about removing the "old" PiiRedaction/SentimentAnalysis